lib/OpenQA/Script/Client.pm

```diff
@@ -153,8 +154,8 @@ sub run ($options, $operation, @args) {
         $options->{url} = $url;
         $options->{params} = \%params;
         $options->{params2} = @ARGV;
-        eval { $res = $client->archive->run($options) };
-        die "ERROR: $@ \n", $@ if $@;
+        try { $res = $client->archive->run($options) }
+        catch ($e) { die "ERROR: $e \n", $e }
```

Comment:
I suppose that second argument was a mistake in b5f09f9761f6fad1c9c63c3fba50f2936b38ee8b
```suggestion
        catch ($e) { die "ERROR: $e\n" }
```